packages/simple-schema/lib/content-assist/element-name.js

```diff
@@ -11,7 +13,22 @@ const { difference, map, filter } = require("lodash");
  * @returns {CompletionSuggestion[]}
  */
 function elementNameCompletion(elementNode, xssElement, prefix = "") {
-  const allPossibleSuggestions = map(xssElement.elements, _ => _.name);
+  const match = prefix.match(NAMESPACE_PATTERN);
+  if (!match) {
+    return [];
+  }
+  const namespacePrefix = match[1];
+  const namespace = find(
+    elementNode.namespaces,
+    _ => _.prefix === namespacePrefix
+  );
+  const possibleElements = filter(
+    xssElement.elements,
+    _ =>
+      !_.namespace ||

```

Comment:
avoid boolean conversions, e.g:
- `_.has(_, "namespace")` if the property may not exist.
- `_.namespace === ""` if it may be empty string